docs/kubernetes/ressources/README.md

```diff
@@ -0,0 +1,539 @@
+# K3s on TF Grid
```

Comment:
Is this file still needed now that we have the `Getting started` ? It seems it is a bit of a duplicate

Reply:
right ..deleted